.cppcheck.supp

```diff
@@ -0,0 +1,16 @@
+// remove some checks
+missingInclude
+unusedFunction
+
+// external libraries
+*:external/json.hpp
+*:external/cxxopts.hpp
+
+// opencascade
+*:*.hxx
```

Comment:
this looks a bit too wide for this to be opencascade specific

Reply:
I know... But we are not using `hxx` extension. Should I list all the opencascade headers having an issue? (there are MANY of them)

Reply:
not possible to do something like "*:opencascase/*.hxx" ?

Reply:
`*:*opencascade/*.hxx` seems to work